DIPs/dip-3.md

```diff
@@ -0,0 +1,82 @@
+---
+dip: 3
+title: Introduce EIP-1559 to Reform the Network Fee Market
+authors: Darwinia Network
+discussions-to: https://github.com/orgs/darwinia-network/discussions/1163
+status: Final
+type: Economic
+created: 2023-12-14
+---
+
+
+# DIP-3
+## Abstract
+This DIP suggests implementing `EIP-1559` within the Darwinia Network to transform its fee structure.
+
+Under this proposal, all transaction fees would be burned, while tips would be awarded directly to block authors.
+This aligns with our future inflation strategy, motivates collators, and paves the way for practical applications of *RING* and *KTON* tokens.
```

Comment:
Because current token mint is based on target supply function every year, after introduce EIP1559, the token mint function need to be changed to incremental function yearly. Otherwise, the burned tokens will be re-minted again based on current target supply function.

Aka. switching from the red one (total supply) to the green one (Issuable this year)

<img width="649" alt="image" src="https://github.com/darwinia-network/DIPs/assets/1070122/38a63818-f42e-4919-b997-f6689d267847">

Reply:
I guess that will be new DIP? Is there any discussion about the new inflation curve?

Reply:
which one will be the first DIP to implement? this DIP or the new one?